client/lomas_client/client.py

```diff
@@ -113,11 +153,37 @@ def get_dummy_dataset(
 
         if res.status_code == HTTP_200_OK:
             data = res.content.decode("utf8")
-            df = pd.read_csv(StringIO(data))
+            dtypes = self.get_df_dtypes()
+            df = pd.read_csv(StringIO(data), dtype=dtypes)
             return df
         print(error_message(res))
         return None
 
+    def get_dummy_lf(
+        self, nb_rows: int = DUMMY_NB_ROWS, seed: int = DUMMY_SEED
+    ) -> Optional[pl.LazyFrame]:
+        """
+        Returns the polars LazyFrame for the dummy dataset with
+        optional parameters.
+
+        Args:
+            nb_rows (int, optional): The number of rows in the dummy dataset.
+
+                Defaults to DUMMY_NB_ROWS.
+
+            seed (int, optional): The random seed for generating the dummy dataset.
+
+                Defaults to DUMMY_SEED.
+
+        Returns:
+            Optional[pl.LazyFrame]: The LazyFrame for the dummy dataset
+        """
+        dummy_pandas = self.get_dummy_dataset(nb_rows=nb_rows, seed=seed)
+
+        if dummy_pandas is None:
+            return None
```

Comment:
why would it be none ?

Reply:
self.get_dummy_metadata fails it does not throw an exception but prints an error message and returns None. We might want to modify this behaviour in the client by throwing the custom exceptions we define in the server.